CHANGELOG.md

```diff
@@ -26,8 +26,6 @@ After this we will switch probably to real [Semantic Versioning 2.0.0](http://se
 * Fix #1732: Correctly replacing template placeholders in helm chart
 * Fix #1734: Add logging during jib builds
 * Fix #1696: Fix put operations for ImageStreams in ApplyService
-* Fix #1738: JibBuildConfiguration ignores BuildImageConfiguration.registry
```

Comment:
Why? It's part of an already delivered version.

Reply:
It's because of this revert https://github.com/fabric8io/fabric8-maven-plugin/pull/1766/commits/c9622553c61dee1d107439f41fb5ca6928427920

Reply:
But that is part of the delivered version in 4.7.1 with this change, the changelog is misleading and incorrect.
If this is part of a commit revert what you should actually do is add a **new** changlog line in the current version indicating `Revert #1738: balblabla` Revert # 1737: reasons to revert`